web/src/views/DandisetLandingView/DandisetActions.vue

```diff
@@ -37,13 +37,15 @@
           </template>
         </DownloadDialog>
       </v-row>
-      <v-row no-gutters>
+      <v-row
+        no-gutters
+      >
         <CiteAsDialog>
           <template
             #activator="{ on }"
           >
             <v-btn
-              id="download"
+              id="cite_as"
```

Comment:
What happened here? Was this wrong before?

Reply:
Yes. I included it in this PR because I am adding to the Dandiset actions.